src/main/java/no/nav/veilarbperson/client/pdl/GqlArguments.java

```diff
@@ -4,25 +4,25 @@
 import lombok.Data;
 import no.nav.common.types.identer.Fnr;
 
-public class PdlPersonVariables {
+public class GqlArguments {
```

Comment:
Tenker heller at `GqlVariables` er et bedre navn. Ordet "variables" er brukt i GQL for å beskrive ting som man sender inn, så tenker det er greit å beholde navnet istedenfor å bruke "arguments".